### ISSUE TITLE
"async.write.enabled=true" for client cache has bad performance compared to disabling it

### ISSUE THREAD

**Issue by secfree** [Alluxio/alluxio#17689]
**Alluxio Version:**
2.9.x

**Describe the bug**
Please refer to https://github.com/Alluxio/alluxio/issues/17134#issuecomment-1606878343 for details

**Expected behavior**
It should have better performance compared to disabling it.

**Urgency**
Normal

**Are you planning to fix it**
Yes

**Comment by secfree** [comment]
cc @beinan

**Comment by LuQQiu** [comment]
@secfree is it because more threads are created for the sync write?
@secfree what kind of data are you trying to read? large file or small file?
what's the concurrency and what's the CPU number?

**Comment by secfree** [comment]
As the screenshot I shared in the refer link

![](https://user-images.githubusercontent.com/7149512/248717195-95e07c52-8ea2-49b8-8ff7-48c6e4b6b1ea.png)

\> what kind of data are you trying to read? large file or small file?
The data is 1 file with 7G size

\>  is it because more threads are created for the sync write?
Not sure, need to investigate more.

\> what's the concurrency and what's the CPU number?
The client is reading with 1 thread. CPU number is 10+

I will do more tests and investigate it.

**Comment by secfree** [comment]
I tested more cases with "async write", including

- Cache data at client side in HDD/RAM
- Read multi files with multi threads at the same time
- Use multiple cache directories in different HDD disks
- ...

but no one showed that it had advantage in performance.

I also profiled the process with Flame Graph, but no obvious bottlenecks found.

Then I consulted @beinan about this feature. He feedback that some other users also tested this but also no positive results reported.

As "async.write.enabled=true" is not a recommended option, I will just ignore this issue.
